### DISCUSSION TITLE
File Icons GitHub

### DISCUSSION THREAD

**Discussion by hellookii** [community/community#44927] [Repositories] [answered]
### Select Topic Area

Product Feedback

### Body

File icons are now very common, but I'm surprised GitHub does not support it. I personally use [an extension](https://github.com/Claudiohbsantos/github-material-icons-extension) to beautify the UI, but I think that a native support would be great.
Below is an image of the UI with extension enabled (image from extension's repo)
![Screenshot (87)](https://user-images.githubusercontent.com/96133536/213808197-7846237e-1be1-40c0-acaf-ac5d4357d65a.png)

**Comment by jge162** [comment, marked as answer]
looks nice

**Reply by hellookii** [reply]
yaa maybe, it's good but nothing more hehe
